setup.py

```diff
@@ -30,7 +30,7 @@
   --int64                                Enable 64bit integers [default: False].
   --omp                                  Enable OpenMP parallelization [default: False].
   --python=<PYTHON_INTERPRETER>          The Python interpreter (development version) to use. [default: ''].
-  --fbindings=<ENABLE_Fortran_API>       Enable compilation of Fortran 90 API bindings <ON/OFF> [default: ON].
+  --fbindings=<TEST_Fortran_API>         Enable compilation of Fortran 90 API bindings <ON/OFF> [default: ON].
```

Comment:
Change to `Test the Fortran 90 bindings for the API.`?

Reply:
Done :)